### PR TITLE
Updated format of note (fixes #2081)

### LINKED ISSUE #2081: Format of a note is incorrect
### Problem
Simple format mistake of a note on http://open-learning-exchange.github.io/#!./pages/vi/vi-github-and-repositories.md

### Steps to reproduce the problem
NA

### Screenshots

noteformatisuue

### Proposed solution
Capitalize "note" to allow the webpage to format it properly

### PR DESCRIPTION
**Pull request by inDepthh**
Fixes #2081 

### Description
Simple capitalization mistake was fixed to allow the page to properly format a note on vi-github-and-repositories.md

### Raw.githack preview link

https://raw.githack.com/inDepthh/inDepthh.github.io/noteIssue/#!pages/vi/vi-github-and-repositories.md

### CODE REVIEW

File: pages/vi/vi-github-and-repositories.md [hunk 1]
```diff
@@ -27,7 +27,8 @@ This is just a summary of the steps that you will need to perform. Please, keep
 * [Sync Your Fork](#Sync_Your_Fork)
 
 
-** Note :** You will see some common names used in git (*which you can change*), such as `master: the name of the main branch`, `upstream: the location where you forked the repository from`, and `origin: the location you cloned repository from`. Both `upstream and origin` are considered **remote**. Keep in mind also, a repository may contain many branches.
+**NOTE**: You will see some common names used in git (*which you can
```
Comment (lmmrssa) [inline pages/vi/vi-github-and-repositories.md]: you break line so please fix this

Comment (inDepthh) [reply pages/vi/vi-github-and-repositories.md]: I believe you mean that I need to delete line break above the change?

Comment (inDepthh) [reply pages/vi/vi-github-and-repositories.md]: It does show up well on the link (raw.githack)

Comment (inDepthh) [reply pages/vi/vi-github-and-repositories.md]: @lmmrssa Ah, sorry for some reason I didn't see that break. I thought you meant the one above the note.

**Pull request merged**